app/assets/stylesheets/themes/scinote.scss

```diff
@@ -469,8 +469,10 @@ a[data-toggle="tooltip"] {
 }
 
 .user-statistics {
+  margin-top: 1em;
+
   .list-inline {
-    margin-left: 15px;
+    margin-left: 0;
```

Comment:
Can we remove this? Or we override some default value with 0?

Reply:
We override with 0